docs/meshstack.aws.metering.md

```diff
@@ -30,3 +32,223 @@ In other words, `Credit`, `Refund`, `RIFee`, `Tax` `SavingsPlanUpfrontFee`, `Sav
+        }
+    ]
+}
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Role
+
+The 3 policies in the [previous section](#policies) should be attached to this role.
+
+In order for the Metering IAM user to assume this role, following trust policy is required:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::123456789123:user/Metering"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {}
+    }
+  ]
+}
+```
+
+## Configuration Reference
+
+This section describes the configuration of a AWS Platform Instance in the meshStack [configuration model](./meshstack.configuration.md)
+at `mesh.platforms` for AWS metering.
+
+<!--snippet:mesh.platforms.aws.kraken#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let AwsPlatformKrakenConfiguration =
+      { platform :
+          {- The meshPlatform identifier -}
+          Text
+      , region :
+          {- The region of the IAM user -}
+          Text
+      , meshfed-service-user :
+          {-
+              Metering IAM user credentials
+                  access-key: "<AWS_ACCESS_KEY>"
+                  secret-key: "<AWS_SECRET_KEY>"
+          -}
+          { access-key : Secret, secret-key : Secret }
+      , cost-and-usage-report :
+          {-
+              Parameters required to process AWS cost and usage reports
+                  name: The name of the report
+                  bucket-name: The name of the S3 bucket where the report is written
+                  bucket-region: The region of the S3 bucket
+                  report-key-prefix: The report path prefix. Generated reports will be located at <bucket-name>/<report-key-prefix>
+          -}
+          { name : Text
+          , bucket-name : Text
+          , bucket-region : Text
+          , report-key-prefix : Text
+          }
+      , organization-root-account-role :
+          {- The role assigned to the metering IAM user -}
+          Text
+      , organization-root-account-external-id :
+          {- The external ID passed to "sts:AssumeRole" when used in privileged context -}
+          Optional Text
+      , cost-item-write-batch-size :
+          {- Cost items written per batch -}
+          Natural
+      , max-report-file-processing-retries :
+          {- The maximum number of attempts to process the report file -}
+          Natural
+      , report-file-process-retry-delay-millis :
+          {- Time in milliseconds before an operation (report file processing) is retried -}
+          Natural
+      , max-bulk-insert-retries :
+          {- The maximum number of attempts to process a cost item batch -}
+          Natural
+      , bulk-insert-retry-delay-millis :
+          {- Time in milliseconds before an operation (cost item batch processing) is retried -}
+          Natural
+      , cost-item-read-page-size :
+          {- Cost items retrieved per query -}
```

Comment:
```suggestion
          {- Cost items to read per query when generating tenant usage reports. This setting is useful to prevent read timeout errors in the case of accounts with a large number of cost items. -}
```